.github/workflows/check-format-black.yml

```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Check Black formatting
+      run: echo Hello, world!
```

Comment:
Do you need this step?

Reply:
I would have thought it would have been nice for the script to be polite 😆 , but it's probably not necessary
I just blindly copy-pasted this, will be correcting it tonight